.gitlab-ci.yml

```diff
@@ -418,14 +418,23 @@ doc:refman:
   needs:
     - build:base
 
-doc:refman:dune:
+doc:refman-html:dune:
   extends: .dune-ci-template
   variables:
     DUNE_TARGET: refman-html
   artifacts:
     paths:
       - _build/log
-      - _build/default/doc/sphinx_build/html
+      - _build/default/doc/refman-html
+
+doc:refman-pdf:dune:
```

Comment:
Do think it is worth the overhead to split the refman generation in two?

Reply:
Basically, nothing is shared between the two targets, and they take quite long. I wouldn't want to delay the generation of the HTML artifact just to ensure that the PDF version builds. That's why I split this in two.

Reply:
I thought they would share some Sphinx databases, etc... The build didn't take long in my machine but it was using -j 4 tho.

Reply:
I tried to write the rules differently so that the cache would be shared but it didn't seem to work. Maybe there is something that I don't understand here though. I wonder if @cpitclaudel would have some insight for this.

Reply:
Even when I try to build the same target twice I don't get cache reuse, because I see `[config changed ('coq_config')]`.  I wonder why that is.

Reply:
I dunno, `coq_config` is generated by configure; but configure should not be called again in a rebuild.

If you call configure it will indeed add the build date etc... this is already scheduled to be fixed by moving to the `Build_info` lib.

Reply:
Got it, I think. Let me prepare a PR

Reply:
I think this PR should help: https://github.com/coq/coq/pull/11892

Reply:
(as long as the two builds share the same cache, of course)

Reply:
> I dunno, coq_config is generated by configure; but configure should not be called again in a rebuild.

That's not the issue: the issue is that we put the whole Python module as a cache key, and equality on that isn't stable.